src/main/java/org/opensearch/security/rest/DashboardsInfoAction.java

```diff
@@ -59,19 +59,19 @@
 
 public class DashboardsInfoAction extends BaseRestHandler {
 
-    private static final List<Route> routes = ImmutableList.<Route>builder()
+    private static final List<ReplacedRoute> replacedRoutes = ImmutableList.<ReplacedRoute>builder()
```

Comment:
I would expect that `/_opendistro/_security/kibanainfo` would be replaced with `/_plugins/_security/dashboardsinfo` both for `GET` and `POST`.

Reply:
I printed out the routes that are created, here are the routes:
```
/_plugins/_security/dashboardsinfo
/_plugins/_security/dashboardsinfo
/_opendistro/_security/kibanainfo
/_opendistro/_security/kibanainfo
```
This is what we expect, as for `dashboardsinfo` in only works with `_plugins` and for `kibanainfo` it only works with `_opendistro`. As you see in routes, I am passing empty for old route for both `kibanainfo` and `dashboardsinfo`:
```
private static final List<ReplacedRoute> replacedRoutes = ImmutableList.<ReplacedRoute>builder()
        .addAll(addRoutesPrefix(
            ImmutableList.of(
                new Route(GET, "/dashboardsinfo"),
                new Route(POST, "/dashboardsinfo")
            ),
            "/_plugins/_security", ""))
        .addAll(addRoutesPrefix(
            ImmutableList.of(
                new Route(GET, "/kibanainfo"),
                new Route(POST, "/kibanainfo")
            ),
            "/_opendistro/_security", ""))
        .build();
```

Reply:
After discussing it with @hsiang9431-amzn , I fixed it in the latest commit